anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -126,14 +126,16 @@ class VulnerabilityMatchTrigger(BaseTrigger):
     fix_available = BooleanStringParameter(
         name="fix_available",
         example_str="true",
-        description="If present, the fix availability for the vulnerability record must match the value of this parameter.",
+        description="If present, the fix availability for the "
```

Comment:
Unrelated change probably caused by pycharm default linter, rollback and use black to format to minimize the changes